lib/std/os/uefi/protocols.zig

```diff
@@ -43,3 +43,8 @@ pub usingnamespace @import("protocols/udp6_protocol.zig");
 pub const hii = @import("protocols/hii.zig");
 pub usingnamespace @import("protocols/hii_database_protocol.zig");
 pub usingnamespace @import("protocols/hii_popup_protocol.zig");
+
+comptime {
```

Comment:
there is another instance of this pattern here

Reply:
Whoops, missed these